README.md

```diff
@@ -22,11 +22,13 @@ jobs:
       - uses: TimonVS/pr-labeler-action@v3
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          CONFIG_FILENAME: ".github/pr-labeler.yml" # optional, .github/pr-labeler.yml is the default value
```

Comment:
Instead of passing it as an env variable I'd like to use the `with` parameter (https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idstepswith). 
We can also add validation for this in `action.yml`: https://help.github.com/en/articles/metadata-syntax-for-github-actions#inputs.

See actions/labeler for an example (https://github.com/actions/labeler/blob/master/src/main.ts#L9). We should also do the same for `GITHUB_TOKEN` but we can do that in another PR :)